data/registry/instrumentation-js-fastify-otel.yml

```
@@ -0,0 +1,20 @@
+# cSpell:ignore fastify
+title: Fastify Instrumentation Library
+registryType: instrumentation
+language: js
+tags:
+  - Node.js
+  - fastify
+license: MIT
+description: Fastify OpenTelemetry auto-instrumentation library.
+authors:
+  - name: Fastify
+     url: https://github.com/fastify
```

Extra space renders the YAML invalid and breaks the build, here's a fix:

```suggestion
authors:
  - name: Fastify
    url: https://github.com/fastify
```